src/components/JoinUs/Learning.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { Padding, Margin } from 'styled-components-spacing'
+import styled from 'styled-components'
+import { Row, CompensatedRow, CompensatedCol } from '../grid'
+import Image from '../Common/Image'
+import { H2, Paragraph } from '../Typography'
+
+import {
+  Section,
+  FirstColumn,
+  SecondColum,
+  Separator,
+  List,
+  Hr,
+  TitleAndBody
+} from './elements'
+
+const InsightAnchor = styled.a`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+`
+
+const Learning = ({ data: { title, text, featuredInsights, list } }) => (
+  <Section greyBg>
+    <Row>
+      <FirstColumn>
+        <H2>{title}</H2>
+      </FirstColumn>
+      <SecondColum width={[1, 1, 1, 1, 6 / 12, 5 / 12]}>
+        <List list={list} />
+      </SecondColum>
+    </Row>
+    <Separator />
```

Comment:
I'm not sure that we need a new `Separator` component.

I think [the padding on Separator](https://github.com/yldio/yld.io/pull/80/files#diff-68a74b60a6cd09266fc4ff3341c7a014R90) belongs on the [`TitleAndList` component](https://trello.com/c/ycIBI03f/277-titleandlist-component) - i.e. on the `Row` here (line 27)

Reply:
From the ticket and designs, I don't feel like this needs to be on that component..what makes you think that?
(Also, if we decided that it does make part of the component, wouldn't this code be removed when that component is done?)

Reply:
That call really help understand this (I thought you were saying that the Hr belong on the `TitleAndList` comp 😃 
I've just created a `TitleAndList` component that can be extracted in the future, that applies that correct padding. I'm not extracting it already since it depends on those `Compensated` grid elements - but at least I think part of the job of creating that compo is done :)